quint/cli-tests.md

```diff
@@ -381,3 +381,9 @@ Regression test for [#1108](https://github.com/informalsystems/quint/issues/1108
 
 <!-- !test check 1108 -->
     quint parse testFixture/_1052one.qnt
+
+### FAIL on run river::noSolution
+
+<!-- !test exit 1 -->
+<!-- !test check parse testFixture/_1060case.qnt - parse case sensitive filenames -->
+    quint parse testFixture/_1060case.qnt
```

Comment:
Should we also check if the error matches the expected? I think this test would pass even before your change.

Reply:
I also wanted to do that. But it's a bit hard to do, since the `quint parse` outputs absolute paths.

Reply:
I've fixed this test by moving it to `io-cli-tests.md` and comparing the outputs.